<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>22</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TableUi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFFFFF;color:#565656</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Home">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #FFFFFF; padding:5px;font: 20pt &quot;Assistant&quot;;</string>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Light">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #FFFFFF; padding:5px; color:#979797;font: 20pt &quot;Assistant&quot;;</string>
         </property>
         <property name="text">
          <string>Light</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Alarm">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #FFFFFF; padding:5px; color:#979797;font: 20pt &quot;Assistant&quot;;</string>
         </property>
         <property name="text">
          <string>Alarm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Weather">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #FFFFFF; padding:5px; color:#979797;font: 20pt &quot;Assistant&quot;;</string>
         </property>
         <property name="text">
          <string>Weather</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Blind">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #FFFFFF; padding:5px; color:#979797;font: 20pt &quot;Assistant&quot;;</string>
         </property>
         <property name="text">
          <string>Blind</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Exit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #FFFFFF; padding:5px; color:#979797;font: 20pt &quot;Assistant&quot;;</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ClockLabel">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #FFFFFF; padding:5px;font: 20pt &quot;Assistant&quot;;</string>
         </property>
         <property name="text">
          <string>10:30:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="TemperatureButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 63 20pt &quot;Assistant SemiBold&quot;;border-style: solid;border-width: 0px; padding:5px;border-radius: 20; background-color: rgb(255, 51, 51); color: rgb(153,0,0)
</string>
         </property>
         <property name="text">
          <string>Temperature: 29 C 
Humidity: 54%</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="LightButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFAE0D;
font: 63 22pt &quot;Assistant SemiBold&quot;;border-style: solid;border-width: 0px; padding:5px;border-radius: 20; background-color: rgb(255, 235, 177);
</string>
         </property>
         <property name="text">
          <string>On - Lamp</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="AlarmButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Assistant SemiBold</family>
           <pointsize>22</pointsize>
           <weight>7</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;border-width: 0px;border-radius:0;background-color: #C4EBFF; padding:5px; color:#35BCFF; border-radius: 20; font: 63 22pt &quot;Assistant SemiBold&quot;</string>
         </property>
         <property name="text">
          <string>Alarm 
6:00</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QPushButton" name="SleepButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 63 22pt &quot;Assistant SemiBold&quot;;border-style: solid;border-width: 0px; padding:5px;border-radius: 20; 
background-color: rgb(138, 138, 138);
</string>
         </property>
         <property name="text">
          <string>Sleep Mode: Off</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
